sed/core/processor.py

```diff
@@ -178,10 +178,35 @@ def __repr__(self):
             df_str = "Data Frame: No Data loaded"
         else:
             df_str = self._dataframe.__repr__()
-        attributes_str = f"Metadata: {self._attributes.metadata}"
-        pretty_str = df_str + "\n" + attributes_str
+        pretty_str = df_str + "\n" + "Metadata: " + "\n" + self._attributes.__repr__()
         return pretty_str
 
+    def _repr_html_(self):
+        html = "<div>"
+
+        html += (
+            f"<details><summary>Dataframe</summary>{self.dataframe.head()._repr_html_()}</details>"
+        )
+
+        # Add expandable section for dataframe
+        html += f"<details><summary>Dask</summary>{self.dataframe._repr_html_()}</details>"
+
+        # Add expandable section for attributes
+        html += "<details><summary>Metadata</summary>"
+        html += self.attributes._repr_html_()
+        html += "</details>"
+
+        # Add expandable section for plots
+        html += "<details><summary>Plots</summary>"
+        # Something like the event histogram can be added here,
+        # but the method needs to output image/html
+        # self.view_event_histogram(dfpid=2, backend="matplotlib")
+        html += "</details>"
```

Comment:
As this should be fast, again I am not sure how helpful this is, in addition as there is the separate method for it. Wen can discuss, though.

Reply:
I have created another method called overview_panel in comments but can be another issue/PR.